src/imitation/policies/base.py

```diff
@@ -23,18 +24,31 @@ def __init__(self, observation_space: gym.Space, action_space: gym.Space):
             action_space=action_space,
         )
 
-    def _predict(self, obs: th.Tensor, deterministic: bool = False):
+    def _predict(
+        self,
+        obs: Union[th.Tensor, Dict[str, th.Tensor]],
```

Comment:
Do we need to change this? `predict` already supports dict-typed observations: https://github.com/DLR-RM/stable-baselines3/blob/master/stable_baselines3/common/policies.py#L323

``` 
observation: Union[np.ndarray, Dict[str, np.ndarray]],
```

I think the way this works in SB3 is that the feature extractor extracts all the values from the dict and concatenate them together for feeding into neural network. We don't want that but I wonder if we can just operate at the level of abstraction of `predict` and be OK?